### DISCUSSION TITLE
如何修改扉页内容：添加新行，删除行和修改内容

### DISCUSSION THREAD

**Discussion by CrazyElements** [sjtug/SJTUThesis#929] [提问] [answered]
在SJTUThesis v1.1 中，如何添加新行，删除行和修改内容

1. 将"论文题目"修改为"题目"
2. 在“论文题目”下一行添加：评分____
![image](https://user-images.githubusercontent.com/105211051/232962751-fd81e2c6-3311-40e2-a8df-951d5cfd2025.png)
即修改为![image](https://user-images.githubusercontent.com/105211051/232962711-c140350f-05b7-4a1f-9dbe-30d8f2bd8e6b.png)
3. 删除"指导教师"这行
![image](https://user-images.githubusercontent.com/105211051/232962429-3b74859b-e455-4370-97ce-7c8ff1c3882a.png)

**Comment by LogCreative** [comment, marked as answer]
如果是 `type=course`，在此处修改扉页信息

https://github.com/sjtug/SJTUThesis/blob/700093d999dc6fe212aa280d04aee6c120470f3b/texmf/tex/latex/sjtuthesis/sjtuthesis-undergraduate.ltx#L145-L153

**Reply by CrazyElements** [reply]
修改成功了，谢谢你
